tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36,py37,pypy3,pre-commit
+envlist = py36,py39,pypy3,pre-commit
```

Comment:
Oops, sorry, didn't mean to leave this in. I don't have 3.6 or 3.7 locally.

Reply:
And I didn't realize `tox -e py39` would work without this change, but it seems to.